test/spec.index.js

```diff
@@ -927,6 +940,221 @@ describe('Template Mixins', function () {
+                    }
+                });
+
+                middleware(req, res, next);
+
+                res.locals['checkbox-group']().call(res.locals, 'field-name-1');
+                render.should.have.been.calledWith(sinon.match({
+                    legendClassName: 'abc def'
+                }));
+
+                res.locals['checkbox-group']().call(res.locals, 'field-name-2');
+                render.should.have.been.calledWith(sinon.match({
+                    legendClassName: 'abc def'
+                }));
+            });
+
+            it('uses locales translation for legend if a field value isn\'t provided', function () {
+                var translate = sinon.stub().withArgs('fields.field-name.legend').returns('Field legend');
+                middleware = mixins({
+                    'field-name': {}
+                }, { translate: translate });
+                middleware(req, res, next);
+                res.locals['checkbox-group']().call(res.locals, 'field-name');
+                render.should.have.been.calledWithExactly(sinon.match({
+                    legend: 'Field legend'
+                }));
+            });
+
+            it('uses locales translation for hint if a field value isn\'t provided', function () {
+                var translate = sinon.stub().withArgs('fields.field-name.hint').returns('Field hint');
+                middleware = mixins({
+                    'field-name': {}
+                }, { translate: translate });
+                middleware(req, res, next);
+                res.locals['checkbox-group']().call(res.locals, 'field-name');
+                render.should.have.been.calledWithExactly(sinon.match({
+                    hint: 'Field hint'
+                }));
+            });
+
+            it('doesn\'t add a hint if the hint doesn\'t exist in locales', function () {
+                middleware = mixins({
+                    'field-name': {}
+                });
+                middleware(req, res, next);
+                res.locals['checkbox-group']().call(res.locals, 'field-name');
+                render.should.have.been.calledWithExactly(sinon.match({
+                    hint: null
+                }));
+            });
+
+            describe('multiple options selected', function () {
+                beforeEach(function () {
+                    middleware = mixins({
+                        'field-name': {
+                            options: [{
+                                label: 'Foo',
+                                value: 'foo'
+                            }, {
+                                label: 'Bar',
+                                value: 'bar'
+                            }, {
+                                label: 'Baz',
+                                value: 'baz'
+                            }]
+                        }
+                    });
+                });
+
+                it('marks foo and bar as selected', function () {
```

Comment:
These descriptions don't provide much information. Would something more along the lines of 'sets previously selected options of a group to true'..?